File: Hospital.Api/Hospital.Api/Models/historiamedica.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Hospital.Api.Models
{
    public class historiamedica
    {
        [Key]
        public int idhistoriamedica { get; set; }
        public int idhistorial { get; set; }
        public string fechahistoria { get; set; }
        public string nombreprocedimiento { get; set; }
        public string tratamiento { get; set; }
        public string cedulapersonal { get; set; }
    }
}

File: WEB/HOSPITEC/src/app/models/paciente/historia-medica.ts
export class HistoriaMedica {
  idhistoriamedica: number;
  idhistorial: number;
  fechahistoria: string;
  nombreprocedimiento: string;
  tratamiento: string;
  cedulapersonal: string;

  constructor(id, idh, fecha, proc, trat, ced) {
    this.idhistorial = id;
    this.idhistorial = idh;
    this.fechahistoria = fecha;
    this.nombreprocedimiento = proc;
    this.tratamiento = trat;
    this.cedulapersonal = ced;
  }

}

